Microsoft.Toolkit.Uwp.UI.Media/Effects/TintEffect.cs

```diff
@@ -11,11 +11,17 @@ namespace Microsoft.Toolkit.Uwp.UI.Media.Effects
     /// A tint effect
     /// </summary>
     /// <remarks>This effect maps to the Win2D <see cref="Graphics.Canvas.Effects.TintEffect"/> effect</remarks>
-    public sealed class TintEffect : IPipelineEffect
+    public sealed class TintEffect : IPipelineNode
     {
         /// <summary>
         /// Gets or sets the int color to use
         /// </summary>
         public Color Color { get; set; }
+
+        /// <inheritdoc/>
+        public PipelineBuilder AppendToPipeline(PipelineBuilder builder)
+        {
+            return builder.Tint(Color);
```

Comment:
Should these methods from the builder go in these classes? They could be static if other effects need to call them as well? But then at least all the logic for the core of an effect is in one place? Would make it easier for others to see how they're done as examples if they want to build their own.